script/utils/transferOwnership.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# Default to dry run mode
+dry_run=true
+
+# Check if required environment variables are set
+if [ -z "$TARGET_RPC_URL" ] || [ -z "$OWNER_ADDRESS" ] || [ -z "$UPGRADER_ADDRESS" ] || [ -z "$EORACLE_CHAIN_ID" ] || [ -z "$DEPLOYER_PRIVATE_KEY" ]; then
+    echo "Error: Required environment variables not set"
+    echo "Please set TARGET_RPC_URL, OWNER_ADDRESS, UPGRADER_ADDRESS, EORACLE_CHAIN_ID, and DEPLOYER_PRIVATE_KEY"
+    exit 1
+fi
+
+# Add after line 4 (after dry_run=true):
+if [ "${dry_run}" = "true" ]; then
+    echo "Running in dry run mode - no transactions will be broadcast"
+fi
+
+# Get chain ID
+CHAIN_ID=$(cast chain-id -r $TARGET_RPC_URL)
+echo "Chain ID: $CHAIN_ID"
+
+# Function to get config file path
+get_config_file() {
+    local chain_id=$1
+    local config_file="script/config/$chain_id/$EORACLE_CHAIN_ID/targetContractAddresses.json"
+    if [ -f "$config_file" ]; then
+        echo "$config_file"
+        return 0
+    fi
+    echo ""
+}
+
+# Get config file path
+CONFIG_FILE=$(get_config_file $CHAIN_ID)
+
+if [ -z "$CONFIG_FILE" ]; then
+    echo "Error: No config file found for chain ID $CHAIN_ID"
+    exit 1
+fi
```

Comment:
remove the internal function
```suggestion
CONFIG_FILE="script/config/$chain_id/$EORACLE_CHAIN_ID/targetContractAddresses.json"

if [ !  -f "$CONFIG_FILE" ]; then
    echo "Error: No config file ${CONFIG_FILE} found for chain ID ${CHAIN_ID}"
    exit 1
fi
```

Reply:
fixed